app/assets/javascripts/articles.js

```diff
@@ -1,13 +1,20 @@
 App.articleQueue.onIncoming = function(data) {
-  var alertDiv = $('#floating-alert');
+  var alertDiv = document.getElementById('#floating-alert');
+  var body = document.querySelectorAll('body');
```

Comment:
Is there a specific reason to use `querySelectorAll` instead of `querySelector` since there is only one `body` tag in the page?

Reply:
after another look i reckon `document.getElementsByTagName("body")[0]` would be the optimum way to select body.

Reply:
Will `document.body` get you what you need?